test/test_preparation_and_conversion.py

```diff
@@ -306,6 +306,21 @@ def cutout_era5(tmp_path_factory):
     return cutout
 
 
+@pytest.fixture(scope="session")
+def cutout_mixed_era5_and_era5t(tmp_path_factory):
+    # Try just getting Berlin
+    # Try just requesting era5 and era5t
+    tmp_path = tmp_path_factory.mktemp("era5_era5t")
+    cutout = Cutout(
+        path=tmp_path / "era5_era5t",
+        module="era5",
+        bounds=BOUNDS,
+        time=slice("2022-09-01", "2022-10-31"),
+    )
```

Comment:
Could we do something like this to address the moving target? 
```suggestion
    import datetime
    from datetime import date, timedelta
    today = date.today()
    first = today.replace(day=1)
    last_day_last_month = first - timedelta(days=1)
    first_day_second_last_month = (last_day_last_month - timedelta(days=32)).replace(day=1)

    tmp_path = tmp_path_factory.mktemp("era5_era5t")
    cutout = Cutout(
        path=tmp_path / "era5_era5t",
        module="era5",
        bounds=BOUNDS,
        time=slice(str(first_day_second_last_month), str(last_day_last_month)),
    )
```

Reply:
We could, yeah. I was thinking it would lead to occasional continuous integration failures on the first day of each month, depending on when exactly the CDSAPI swaps `ERA5T` for `ERA5` data. But on second thought - the test would actually not fail in that case. Worst case it would either only contain `ERA5T` or only `ERA5` data - thus temporarily not testing the merging of the 2 datasets, but that's really not a big problem.

Reply:
I updated the test accordingly - but reduced the time span from the full 2 months to 2 days (last of the second-previous month and first of the previous month) to make the CDSAPI request simpler and faster. Please have a look.